app.js

```diff
@@ -2,169 +2,143 @@ const createError = require('http-errors');
 const express = require('express');
 const logger = require('morgan');
 const cors = require('cors');
-const app = express();
+const Sequelize = require('sequelize');
 const parseISO = require('date-fns/parseISO');
 const startOfDay = require('date-fns/startOfDay');
 const endOfDay = require('date-fns/endOfDay');
 const isValid = require('date-fns/isValid');
 
+const app = express();
+
 // Sequelize init
-const Sequelize = require('sequelize');
 const host = process.env.RDS_HOSTNAME || 'localhost';
 const user = process.env.RDS_USERNAME || 'hrm';
 const pass = process.env.RDS_PASSWORD || '';
-const port = process.env.RDS_PORT || '5432';
 const apiKey = process.env.API_KEY;
 
-const Op = Sequelize.Op;
-const version = "0.3.0";
+const { Op } = Sequelize;
+const version = '0.3.0';
```

Comment:
Ah... I roughly try to keep this in sync with flex-plugins whenever we change code here... so let's make this 0.3.6 now.